src/torchcodec/_core/Encoder.cpp

```diff
@@ -568,6 +569,44 @@ AVPixelFormat validatePixelFormat(
   }
   TORCH_CHECK(false, errorMsg.str());
 }
+
+void validateNumericOption(
```

Comment:
For now this expects an `int`, not a float or double. Let's reflect that in the name. We may define this with a template later to be truely about generic numeric options.


```suggestion
void validateIntOption(
```